db/migrate/20201202201111_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[5.1]
+  def change
+    create_table :comments do |t|
+      t.string :comment
+      t.references :users, null: false, foreign_key: true
+      t.references :posts, null: false, foreign_key: true
+      t.timestamps
+    end
+  end
+end
```

Comment:
**same as before**
```ruby
class CreateComments < ActiveRecord::Migration[5.1]
  def change
    create_table :comments do |t|
      t.string :body # OPTIONAL
      # Notice that I rename it to body since it might cause issue on
      # your code when yoou name it as the singular of the current object
      t.references :user, null: false, foreign_key: true
      t.references :post, null: false, foreign_key: true
      t.timestamps
    end
  end
end
```